drafts/zip-daira-disable-sprout-outputs.rst

```diff
@@ -0,0 +1,156 @@
+::
+
+  ZIP: 211
+  Title: Disabling Sprout Outputs
+  Owners: Daira Hopwood <daira@z.cash>
+  Credits: Sean Bowe <sean@z.cash>
+  Status: Draft
+  Category: Consensus
+  Created: 2019-03-29
+  License: MIT
+
+
+Terminology
+===========
+
+The key words "MUST" and "MAY" in this document are to be interpreted as described in
+RFC 2119. [#RFC2119]_
+
+The term "network upgrade" in this document is to be interpreted as described in ZIP 200
+[#zip-0200]_.
+
+The term "Sapling" in this document is to be interpreted as described in ZIP 205
+[#zip-0205]_.
+
+The terms "commitment tree", "treestate", and "dummy note" are to be interpreted as
+described in [#protocol]_.
+
+
+Abstract
+========
+
+This proposal defines a modification to the transaction format to disable outputs to
+Sprout addresses. This takes a step toward being able to remove the Sprout protocol,
+thus reducing the overall complexity and attack surface of Zcash.
+
+
+Motivation
+==========
+
+The first iteration of the Zcash network, called Sprout, provided a shielded payment
+protocol that was relatively closely based on the original Zerocash proposal. [#zerocash]_
+
+The Sapling network upgrade [#zip-0205]_ introduced significant efficiency and
+functionality improvements for shielded transactions. It is expected that over time,
+the use of Sapling shielded transactions will replace the use of Sprout.
+
+The Sapling and Sprout shielded protocols employ different cryptographic designs.
+Since an adversary could potentially exploit any vulnerability in either design,
+supporting both presents additional risk over supporting only the newer Sapling protocol.
+
+For example, a vulnerability was discovered in the zero-knowledge proving system
+originally used by Zcash that could have allowed counterfeiting [#counterfeiting]_.
+While this particular vulnerability was addressed (also for Sprout shielded transactions)
+by the Sapling upgrade, and we are not aware of others at the time of writing, the
+possibility of other cryptographic weaknesses cannot be entirely ruled out.
+
+In addition, the Zcash specification and implementation incurs complexity and
+"technical debt" from the requirement to support both shielded transaction protocols.
+
+Removing the ability to send to Sprout outputs is a first step toward reducing this
+complexity and potential risk. This does not prevent extracting value held in Sprout
+addresses and sending it to transparent addresses, or to Sapling addresses via the
+migration tool [#migration]_.
```

Comment:
Doesn't this ZIP conflict with the Sprout-to-Sapling migration tool, which does send a (random) amount of change back to the Sprout address?

Reply:
Yes. This has now been fixed.